apps/src/templates/sectionAssessments/MultipleChoiceSurveyQuestionDialog.jsx

```diff
@@ -0,0 +1,99 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import Button from '@cdo/apps/templates/Button';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import i18n from '@cdo/locale';
+import DialogFooter from '@cdo/apps/templates/teacherDashboard/DialogFooter';
+import color from '@cdo/apps/util/color';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import {multipleChoiceDataPropType} from './assessmentDataShapes';
+
+const styles = {
+  dialog: {
+    paddingLeft: 20,
+    paddingRight: 20,
+    paddingBottom: 20
+  },
+  instructions: {
+    marginTop: 20
+  },
+  answers: {
+    float: 'left',
+    width: 550
+  },
+  icon: {
+    color: color.level_perfect
+  },
+  iconSpace: {
+    width: 40,
+    float: 'left'
+  },
+  answerBlock: {
+    width: '100%'
+  },
+  answerLetter: {
+    width: 30,
+    float: 'left',
+    fontWeight: 'bold'
+  }
+};
+
+class MultipleChoiceSurveyQuestionDialog extends Component {
+  static propTypes = {
+    isDialogOpen: PropTypes.bool.isRequired,
+    closeDialog: PropTypes.func.isRequired,
+    questionData: multipleChoiceDataPropType.isRequired
+  };
+
+  render() {
+    const {questionData} = this.props;
+
+    // Questions are in markdown format and should not display as plain text in the dialog.
+
+    return (
+      <BaseDialog
+        useUpdatedStyles
+        isOpen={this.props.isDialogOpen}
+        style={styles.dialog}
+        handleClose={this.props.closeDialog}
+      >
+        <div>
+          <h2>{i18n.questionDetails()}</h2>
+          <div style={styles.instructions}>
+            <SafeMarkdown markdown={questionData.question} />
+          </div>
+          {questionData.answers && questionData.answers.length > 0 && (
+            <div>
+              {questionData.answers.map((answer, index) => {
+                return (
+                  <div key={index} style={styles.answerBlock}>
+                    <div style={styles.answerLetter}>
+                      {answer.multipleChoiceOption}
+                    </div>
+                    <div style={styles.answers} />
+                    <SafeMarkdown
+                      markdown={
+                        answer.text + ' (' + answer.percentAnswered + '%)'
+                      }
+                    />
+                    <div style={{clear: 'both'}} />
+                  </div>
+                );
+              })}
+            </div>
+          )}
+        </div>
+        <DialogFooter>
+          <Button
+            __useDeprecatedTag
```

Comment:
Is it possible to avoid `__useDeprecatedTag`? 
This param causes Button to use a <div> instead of a <button> element.

Reply:
tbh i didn't even bother learning what that tag did, thinking consistency with other similar dialogs was good enough for me. see above about slippery slopes. if @clareconstantine thinks it's worth the time, i'm happy to slide down those slopes and use these two dialogs to develop some higher levels of abstraction on top of `BaseDialog` so we don't have duplicated code in these two components. if not though, i think keeping it consistent with the other dialog would be better than cleaning up just one.

thanks for explaining what this tag does though! i'll need to do some reading to understand the implications, but good to know the gist of it!

Reply:
We generally try to use idiomatic HTML where possible on new elements for accessibility purposes and WCAG compliance. It's an area we have been historically bad at, but via deprecations of various non-compliant elements this year, we are starting to make progress. For this particular deprecated tag, we'd generally want a pretty significant reason to keep it here (clickable divs are a focus for accessibility work this year). If you do decide to leave it, please create an item on the backlog so we can come back and clean it up. 

In general for features/elements/modules we've deprecated, we take a "clean up as you go" approach. It spreads the work out across the entire engineering team so we can eventually fully remove them without a large disruption.

Reply:
I second @jmkulwik here. It'd be good to clean this up. Also, what is the downside to taking the time to find out what all the code you write does? Is there an aggressive deadline for this PR?

Reply:
Also in general I recommend taking time to understand code written for PRs - up to a limit of course, since its generally not possible to understand how every detail of what you write works - but generally speaking, understanding what the parameters used in all of the functions you call do falls within the scope of what I expect someone writing a diff to be responsible for in the large majority of cases. Is there an aggressive deadline on this or other tasks that is causing this to be difficult?

Reply:
right on, i'm more than happy to spend time cleaning this up. and good point about understanding any code that i'm reusing @mvkski – in retrospect this was a very lazy approach. stay tuned!